### PR TITLE
Remove Google Code Reference and Fix Grammar

### PR DESCRIPTION
**Pull request by Pringo**
I'm not familiar enough with this code to know if the rest of the comment is still accurate. Please let me know if anything else needs to be changed.

### CODE REVIEW

**Comment by RisingFog** [comment]
LGTM, perhaps do a quick file scan for any reference to Google code?

File: Source/Core/VideoCommon/DriverDetails.h [hunk 1]
```diff
@@ -74,7 +74,7 @@ namespace DriverDetails
 		// Pinned memory is disabled for index buffer as the AMD driver (the only one with pinned memory support) seems
 		// to be broken. We just get flickering/black rendering when using pinned memory here -- degasus - 2013/08/20
 		// This bug only happens when paired with base_vertex.
-		// Please see issue #6105 on Google Code. Let's hope buffer storage solves this issues.
+		// Please see issue #6105. Let's hope buffer storage solves this issue.
```
Comment (Phatcat) [inline Source/Core/VideoCommon/DriverDetails.h]: Issue 6105 is not open anymore, so the comment altogether seems outdated.

Comment (Pringo) [inline Source/Core/VideoCommon/DriverDetails.h]: Yes I noticed that too but I assume since the code is still in use that the comment should be left.

**Comment by Pringo** [comment]
@RisingFog I looked through the GitHub repository search and I think this is the last file remaining with an outdated Google Code reference to the old site.

**Pull request merged**